### PR TITLE
Fix skytem raw data import

### PR DESCRIPTION
**Pull request by redhog**

### CODE REVIEW

**Comment by mp-emrld** [comment]
Hei,  did the following:
- moved file parser for the SkyTEM/Workbench geometry file (.gex )from emerald-aux-data
- moved file parser for SkyTEMs system response files (.sr2) from emerald-aux-data
- added some example notebooks, that illustrate how to use these file parsers, BUT NO DATA

**Comment by redhog** [comment]
Cool!

One small thing: Please rename functions to follow pep8: this_is_a_function_name() instead of thisIsnTPep8CompliantDontUse()

File: README.md [hunk 1]
```diff
@@ -2,38 +2,7 @@
 
 Parser for the Aarhus Workbench XYZ format as documented in the [Workbench reference](http://www.hgg.geo.au.dk/HGGSoftware/workbench/Workbench_A-Z_reference.pdf) for geophysical data
 
-# Usage
```
Comment (redhog) [inline README.md:5]: Why did you delete the short intro to usage? This hasn't been moved to any of the example notebooks either...

Comment (mp-emrld) [reply README.md:5]: This is by mistake - sorry, just put it back

**Comment by mp-emrld** [comment]
closes https://github.com/emerald-geomodelling/libaarhusxyz/issues/5
closes https://github.com/emerald-geomodelling/libaarhusxyz/issues/6
closes https://github.com/emerald-geomodelling/emerald-aux-data/issues/5

File: libaarhusxyz/gex_read.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            general[key]=general[key][0]
+        if len(general[key])==1:
+            general[key]=general[key][0]
+        else:
+            general[key]=np.array(general[key])
+    return general
+
+
+
+def readfile(filename):
```
Comment (redhog) [inline libaarhusxyz/gex_read.py:67]: rename to parse_gex() for consistency?

File: libaarhusxyz/sr2_read.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+def get_system_response(text):
+    data=[]
+    for lines in text:
+        numberstrings   = lines.split()
+        numbers=[float(s) for s in numberstrings]
+        data.append(numbers)
+    return np.array(data)
+
+
+def readfile(sr2_filename):
```
Comment (redhog) [inline libaarhusxyz/sr2_read.py:34]: rename to parse_sr2() for consistency?

File: libaarhusxyz/__init__.py [hunk 4]
```diff
@@ -34,7 +34,7 @@ def columns_to_layers(columns):
```
Comment (redhog) [inline libaarhusxyz/__init__.py:34]: add

```
from gex_read import parse_gex
from sr2_read import parse_sr2
```

?

Comment (redhog) [reply libaarhusxyz/__init__.py:34]: Maybe actually move all functions to a `xyz_read.py` and add the same kind of import for it?
